doc/api/tls.md

```diff
@@ -2042,6 +2045,17 @@ changes:
     e.g. `0x05hello0x05world`, where the first byte is the length of the next
     protocol name. Passing an array is usually much simpler, e.g.
     `['hello', 'world']`. (Protocols should be ordered by their priority.)
+  * `ALPNCallback(params)`: {Function} If set, this will be called when a
```

Comment:
For naming, `alpnCallback` would be more conventional, also remove the `(params)`

Reply:
This would make it inconsistent with the existing `ALPNProtocols` option though, which feels odd. Is that OK?